node/status_node_services.go

```diff
@@ -94,6 +95,7 @@ func (b *StatusNode) initServices(config *params.NodeConfig, mediaServer *server
 	services = append(services, b.CommunityTokensService())
 	services = append(services, b.stickersService(accDB))
 	services = append(services, b.updatesService())
+	services = append(services, b.connectorService())
```

Comment:
in case we don't want to start provider for mobile i suggest to add a setting :
`services = appendIf(config.ConnectorConfig.Enabled, services, b.connectorService())`